tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+skipsdist = True
+envlist = py27,py35
+[testenv]
+deps = -rrequirements.txt
+commands = coverage run -m unittest discover src/tests
```

Comment:
would expect tox be used in `project_tests.sh` like in https://github.com/elifesciences/elife-tools/blob/develop/project_tests.sh#L4-L7